docs/Setup.md

```diff
@@ -3,6 +3,8 @@ This will walk you through installing HuskSync on your network of Spigot servers
 ## Requirements
 > **Note:** If the plugin fails to load, please check that you are not running an [incompatible version combination](Unsupported-Versions)
 
+> **Note:** If you wish to use MongoDB instead of MySQL look [here](Using-MongoDB)
```

Comment:
Thanks for updating the docs! I think it'd probably be easier (given there's not much different) if we just put the Mongo information on this page under a collapsed segment. (i.e. under Step 3, we just have collapsed `<details>` block segments for MySQL / MariaDB).

Also: Don't forget to update the requirements listed in the README, too!